__tests__/helpers/LocalStorageMock.js

```diff
@@ -4,14 +4,17 @@ const localStorageMock = (function () {
 
     return {
         getItem(key) {
-            return store[key];
+            return store[key] || null;
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem they should return `null` if the key doesn't exist (tested in devtools as well)